chunky/src/java/se/llbit/chunky/renderer/Postprocess.java

```diff
@@ -48,7 +48,7 @@
     }
   };
 
-  public static final Postprocess DEFAULT = GAMMA;
```

Comment:
Why would you change the default value here?

Reply:
ACES Filmic is a much more standard tonemapping curve, gamma correction is just a slightly improved linear which induces a lot of clipping without preserving contrast and should be avoided for HDR to LDR conversions. Here's an example of what a tonemapping curve should look like https://www.shadertoy.com/view/llXyWr compared to gamma correction https://miro.medium.com/max/373/1*5CLa9Ue6U2F0f_EwRtY_pA.jpeg